### DISCUSSION TITLE
What's the difference between system and current brightness?

### DISCUSSION THREAD

**Discussion by DavidLemayian** [aaassseee/screen_brightness#25] [Q&A] [answered]
What's the difference between system and current brightness?

**Comment by aaassseee** [comment, marked as answer]
Hi @DavidLemayian.
System brightness means the brightness on system setting, Current brightness means the application brightness.

**Reply by DavidLemayian** [reply]
Thanks @aaassseee . I think I understand

**Reply by aaassseee** [reply]
Feel free to ask.
